MODULE.bazel

```diff
@@ -0,0 +1,22 @@
+module(
```

Comment:
We should enable a CI run with Bzlmod (only the default in the not yet released Bazel 7).

Reply:
I am not sure CI is running in this repo. I will try to set up a BCR presubmit also.